benchmark/sparse/linear_regression/run_sparse_benchmark.py

```diff
@@ -0,0 +1,39 @@
+import argparse
+
+import keras_tf_model
+import mxnet_sparse_model
+import numpy as np
+from scipy.sparse import csc_matrix
+
+
+def invoke_benchmark(batch_size, mode):
+    # Fix random seed for reproducibility
+    np.random.seed(7)
```

Comment:
Now we can remove this fixed seed since same data is passing to both modes.

Reply:
Done